CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ These are the current rules:
 - MethodSeparator
 - MethodParametersOnNewLine
 - PropertyLiteralCase
+- TwoEmptyLines above a method
```

Comment:
Do we need to add this new feature here? Or we need to add this line to a new version like # v1.4.2?

Reply:
Will it be better to add about new rule into `pslLint/cli/README.md`? What do you think? )

Reply:
Yes, that readme shows up here: https://www.npmjs.com/package/psl-lint

We should keep that one up-to-date.